tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -38,7 +39,16 @@ def add_capacity_test():
     osd_size = storage_cluster.get_osd_size()
     existing_osd_pods = get_osd_pods()
     existing_osd_pod_names = [pod.name for pod in existing_osd_pods]
-    result = storage_cluster.add_capacity(osd_size)
+
+    if ui_add_capacity_conditions():
+        try:
+            result = add_capacity_ui(osd_size)
+        except Exception as e:
+            logging.error(e)
+            result = storage_cluster.add_capacity(osd_size)
+    else:
```

Comment:
maybe worth adding a massage saying that add capacity from UI is not applicable and CLI method will be done

Reply:
done